pendulum_control/include/pendulum_control/rtt_executor.hpp

```diff
@@ -120,6 +123,16 @@ class RttExecutor : public rclcpp::Executor
     return 0;
   }
 
+  void spin_some(std::chrono::nanoseconds max_duration = std::chrono::nanoseconds(0)) override
+  {
+    decorableExecutor->spin_some(max_duration);
```

Comment:
By default, pass a null decorator and check for null in this method and otherwise call the parent class.

Reply:
Done